sdk/keyvault/test-resources.json

```diff
@@ -206,53 +206,53 @@
             "objectId": "[parameters('testApplicationOid')]",
             "permissions": {
               "keys": [
-                "backup",
-                "create",
-                "decrypt",
-                "delete",
-                "encrypt",
-                "get",
-                "getrotationpolicy",
-                "import",
-                "list",
-                "purge",
-                "recover",
-                "restore",
-                "rotate",
-                "sign",
-                "setrotationpolicy",
-                "unwrapKey",
-                "update",
-                "verify",
-                "wrapKey"
+                "Get",
```

Comment:
Same here, why upper case is necessary?

Reply:
Due to the incomplete setting of KeyVault permissions, it is not possible to perform rotation strategy related tests in UsGov and China Cloud. The revised settings are taken from the template data exported from the Azure portal.